src/cascadia/TerminalApp/ColorScheme.idl

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import "../TerminalSettings.idl";
+
+namespace TerminalApp
+{
+    [default_interface] runtimeclass ColorScheme {
+        ColorScheme();
+        ColorScheme(String name, UInt32 defaultFg, UInt32 defaultBg, UInt32 cursorColor);
+
+        void ApplyScheme(Microsoft.Terminal.TerminalControl.IControlSettings terminalSettings);
+
+        String Name { get; };
+
+        UInt32 Foreground { get; };
+        UInt32 Background { get; };
+        UInt32 SelectionBackground { get; };
+        UInt32 CursorColor { get; };

```

Comment:
let's not dig this hole deeper. Introduce a Color struct type.

Reply:
you can specialize its conversion to/from til::color

Reply:
we probably should have just gone with `Windows.UI.Color` - my bad

Reply:
Updated ColorScheme to use Windows.UI.Color. Should I go ahead and propagate that change to `ICoreSettings` too? I'm already here.